CHANGES.md

```diff
@@ -164,6 +164,8 @@ To be released.
     [[#759]]
  -  Fixed a bug where `BlockChain<T>` had rendered and evaluated actions in
     the genesis block during forking.  [[#763]]
+ -  Fixed a bug where `DefaultStore.ForkStateReferences()` forked state reference except
+    the genesis block.  [[#776]]
```

Comment:
Although I am not sure either, this patch does not need to have its changelog, because we've never released `DefaultStore` as a public interface.

@planetarium/libplanet Opinions?

Reply:
I agree.